### PR TITLE
run acceptance tests in zuul job

### PR DESCRIPTION
**Pull request by fredericlepied**

### CODE REVIEW

**Comment by fredericlepied** [comment]
recheck

File: zuul.d/run.yaml [hunk 1]
```diff
@@ -22,3 +22,37 @@
...
+        block: |
+          default:
+            wazo_host: localhost
+            debug:
+              global: True
+              
+              acceptance: True
+              linphone: False  # otherwise, there are unprintable (^H) character that jenkins cannot decode in junit.xml
+            setupd:
+              timeout: 120
```
Comment (fblackburn1) [inline zuul.d/run.yaml]: this timeout is now useless, the default has been increased to 60 seconds (I removed it from jenkins config)

https://github.com/wazo-pbx/wazo-setupd-client/blob/master/wazo_setupd_client/commands/setup.py#L7

**Comment by fredericlepied** [comment]
recheck

**Comment by fredericlepied** [comment]
recheck

**Review by fblackburn1 — APPROVED** [review]

**Comment by fblackburn1** [comment]
regate

**Pull request merged**